src/tools/Repl/Program.cs

```diff
@@ -138,11 +164,23 @@ public static void REPL(TextReader input, bool echo)
                     // variable assignment: Set( <ident>, <expr> )
                     if (TryMatchSet(expr, out var varName, out var varValue))
                     {
-                        Console.WriteLine(varName + ": " + PrintResult(varValue));
+                        Console.WriteLine(varName + ": " + PrintResult(varValue));
+                        output?.WriteLine(varName + ": " + PrintResult(varValue));
+                    }
+
+                    // IR pretty printer: IR( <expr> )
+                    else if ((match = Regex.Match(expr, @"^\s*IR\((?<expr>.*)\)\s*$", RegexOptions.Singleline)).Success)

```

Comment:
Nope, don't need to.  It's a pseudo function that always returns true, it can be restricted to just one at a time.  We don't support the chaining operator with the Set function either, for example `Set(x,3); Set(y,5)` is not supported.